src/popup/router/components/Filters.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <div
-    class="filters"
     data-cy="filters"
-    :class="{ fixed }"
+    :style="cssVars"
+    :class="['filters', { sticky, 'with-search': withSearch }]"
```

Comment:
I suggest to not merge the static CSS classes with dynamic ones. I know this is tempting to have only one line if we have only one static class. But it would look really bad if there would be more static classes.